test/testdata/envoy.yaml

```diff
@@ -100,7 +100,7 @@ data:
                           grpc_service:
                             envoy_grpc:
                               cluster_name: ext_proc
-                              authority: vllm-llama3-8b-instruct-epp.default:9002
+                              authority: vllm-llama3-8b-instruct-epp.inf-ext-e2e:9002
```

Comment:
I would use a NAMESPACE env var with a default value (in the Makefile)

Reply:
Ah ok! I'll give that a try.